src/GameBoard.js

```diff
@@ -22,4 +22,13 @@ GameBoard.prototype.switchMarker = function(playerMarker) {
   return playerMarker;
 }
 
+GameBoard.prototype.chooseAvailableSpace = function() {
+  for (i = 0; i <= 8; i++) {
+    var num = this.gameBoard[i].match(/[0-8]/);
```

Comment:
Also not directly related to this PR, but very important (and something to think about before you dive to deep into minimax)... Does it make sense for your game board to look like ['0','1','2','3',4','5','6','7','8']? That seems like it _MIGHT_ make sense in a console version of the game (where the number is used to reference the spot you want to go in), but when you're using a UI like this there's no reason to map the index of each empty spot to the empty spot. Think about using something else to represent nothing being there!